pkg/storaged/stratis-details.jsx

```diff
@@ -703,6 +788,33 @@
                         unlock_with_keydesc(key_desc);
                 }));
     }
+
+    if (stopped_props.key_description &&
+        stopped_props.key_description.t == "(bv)" &&
+        stopped_props.key_description.v[0]) {
+        if (stopped_props.key_description.v[1].t != "(bs)" ||
+            !stopped_props.key_description.v[1].v[0]) {
+            dialog_open({
+                Title: _("Error"),
+                Body: _("This pool can not be unlocked here because its key description is not in the expected format.")
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230525-101055-c79bf30c-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)